lib/src/excel.dart

```diff
@@ -33,20 +40,26 @@ class ExcelDart {
     'End Date'
   ];
 
-  /// Waits for Office APIs to be ready and then populates the spreadsheet with
+  /// Waits for Office APIs to be ready and then creates
+  /// a new spreadsheet with [sheetName] and populates the spreadsheet with
   /// entries in the [insertionOrderList]
-  static void populate(List insertionOrderList) async {
+  static void populate(List<InsertionOrder> insertionOrderList) async {
     await Office.onReady(allowInterop((info) async {
       await ExcelJS.run(allowInterop((context) {
-        // Grabs the current worksheet
-        final sheet = context.workbook.worksheets.getActiveWorksheet();
+        // Adds a new worksheet
+        final sheet = context.workbook.worksheets.add('Query');
```

Comment:
Each new sheet needs a unique name. New issue created for this #36, and will be addressed in a separate PR.